pkg/operator/ceph/file/subvolumegroup/controller_test.go

```diff
@@ -243,15 +244,18 @@ func TestCephClientController(t *testing.T) {
 
 	t.Run("success - external mode csi config is updated", func(t *testing.T) {
 		cephCluster.Spec.External.Enable = true
+		csiOpClientProfile := &csiopv1a1.ClientProfile{}
+
+		s.AddKnownTypes(cephv1.SchemeGroupVersion, &cephv1.CephBlockPoolList{}, &csiopv1a1.ClientProfile{})
```

Comment:
Can we make these unit test changes upstream? The unit test could enable the csi operator for these individual unit test cases. Then no need for this difference downstream, it would just be purely the `if` conditions.